spec/system/user/users_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe 'Index users page', type: :system do
```

Comment:
- [ ] **[OPTIONAL]** I think file should be renamed `index_spec.rb` as that is more intuitive than `users_spec.rb`

Reply:
Excellent, I will rename that